Rectangle/Snapping/SnappingManager.swift

```diff
@@ -405,6 +405,27 @@ class SnappingManager {
         
         if loc.y >= frame.minY && loc.y < frame.minY + marginBottom && !ignoredSnapAreas.contains(.bottom) {
             let thirdWidth = floor(frame.width / 3)
+            let quarterWidth = floor(frame.width / 4)
+            
+            // check first quarter column
+            if Defaults.quarterColumnsSnapAera.userEnabled {
+                if loc.x >= frame.minX && loc.x <= frame.minX + quarterWidth {
+                    if Defaults.eightsSnapArea.userEnabled {
```

Comment:
Quarter columns would need to be enabled in order for eighths to be applied. It seems like it would be more desirable to have these be configurable separate from each other?